landavailability/api/serializers.py

```diff
@@ -1,8 +1,38 @@
 from .models import BusStop
 from rest_framework import serializers
+from django.contrib.gis.geos import GEOSGeometry
+import json
 
 
 class BusStopSerializer(serializers.ModelSerializer):
+    srid = serializers.IntegerField(write_only=True)
+
     class Meta:
         model = BusStop
         fields = '__all__'
+        extra_kwargs = {
```

Comment:
Just in the interests of clarity, is it worth a comment here explaining why the validator for amic_code is removed?